index.js

```diff
@@ -1 +1,17 @@
-module.exports = require('./src/parse');
+var peliasConfig = require( 'pelias-config' ).generate();
+
+var text_analyzer = peliasConfig.api.textAnalyzer || 'addressit';
+
+// Changes to incorporate libpostal were made to minimize the impact to pelias-api
+// so exports cannot be a function to conditionally return an analyzer instance.
+// This can be changed when addressit goes away and libpostal has become the norm.
+if ('libpostal' === text_analyzer) {
+  var postal = require('node-postal');
+  module.exports = require('./src/libpostalParser').create(postal.parser.parse_address);
+}
+else if ('addressit' === text_analyzer) {
+  module.exports = require('./src/addressItParser');
+}
+else {
+  throw 'Unsupported analyzer \'' + text_analyzer + '\', valid values are \'addressit\' and \'libpostal\'';
```

Comment:
Sorry to be difficult but node exceptions need to be instances of the Error class like the following in order for the stack to be presented along with the error. 

throw new Error( 'message' );

Thanks! :rainbow: